server/docs/source/index.html.md

```diff
@@ -1426,6 +1426,79 @@ Create an external registry.
 
 `DELETE /v1/grids/{grid_id}/external_registries`
 
+# Domain Authorizations
+
+Let's Encrypt domain authorization management for certificate handling.
+
+## Domain authorization
+
+```json
+{
+  "domain": "web.52.57.23.91.xip.io",
+  "challenge": {
+    "token": "4caq-Qll7ksacYp0Wd_eW4VTQlLp28ODKfBWZrzNbeU",
+    "uri": "https://acme-staging.api.letsencrypt.org/acme/challenge/0GL_fUeZGnbxilMGbcUoAS3UOz_dK8q4TdxDgeCrOs0/55065930",
+    "type": "tls-sni-01"
+  },
+  "challenge_opts": null,
+  "authorization_type": "tls-sni-01",
+  "linked_service": "e2e/null/lb",
+  "service_deploy_id": "599eac26af8cd3000995b36d"
```

Comment:
Out of date.

Reply:
I'll fix the api docs once we settle for some api design :)

Reply:
I think the API is okay now